extension/lib/main-frame.coffee

```diff
@@ -29,7 +29,10 @@ module.exports = ->
 
   loadConfig = ->
     configDir = prefs.get('config_file_directory')
-    scope = {vimfx: createConfigAPI(vim, onShutdown)}
+    scope = {
+      vimfx: createConfigAPI(vim, onShutdown)
+      content: content
```

Comment:
Instead of `content: content` you could also write just `content` :)

Reply:
Done.